pages.fr/common/ansible-playbook.md

```diff
@@ -0,0 +1,28 @@
+# ansible-playbook
+
+> Exécute les taches définies dans le playbook sur les machines distantes via SSH.
+> Plus d'informations : <https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html>.
+
+- Exécute les taches d'un playbook :
```

Comment:
I just realized the difference between `tâche` for "task" and `tache` for  "dirty mark". 🤓 
See https://dictionnaire.lerobert.com/definition/tache

```suggestion
- Exécute les tâches d'un playbook :
```